src/NServiceBus.Core/Serializers/XML/XmlDeserialization.cs

```diff
@@ -629,17 +629,21 @@ object GetPropertyValue(Type type, XmlNode n)
 
         PropertyInfo GetProperty(Type t, string name)
         {
-            IEnumerable<PropertyInfo> props;
-            cache.typeToProperties.TryGetValue(t, out props);
+            IEnumerable<PropertyInfo> properties;
+            if (!cache.typeToProperties.TryGetValue(t, out properties))
```

Comment:
ah, saw now that the other changes where in some tests. I can live with that :+1: